tests/ffi/tests.cc

```diff
@@ -206,6 +206,16 @@ void c_take_ref_rust_vec(const rust::Vec<uint8_t> &v) {
   }
 }
 
+void c_take_ref_rust_vec_copy(const rust::Vec<uint8_t> &v) {
+  // Test vector copy which was not compiling under gcc 7
+  std::vector<uint8_t> cxx_v;
+  std::copy(v.begin(), v.end(), back_inserter(cxx_v));
+  uint8_t sum = std::accumulate(cxx_v.begin(), cxx_v.end(), 0);
+  if (sum == 200) {
+    cxx_test_suite_set_correct();
+  }
```

Comment:
Let's add a call from tests/test.rs so this assertion is actually executed.

Reply:
Resolved in ba22ba8